evals/results/baseline/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "testdata_path": "ground_truth.jsonl",
-    "results_dir": "results/gpt-4o-mini",
+    "results_dir": "results/experiment<TIMESTAMP>",
```

Comment:
does this get replaced automatically?

Reply:
Oh yes when I ran this eval, I didnt bother changing the results_dir, so thats why we see it changed in the config back to the default configuration. I renamed the folder after the fact instead. I could pretend that I had specified a folder name, but this config.json does actually reflect how I run it, so its true.

Reply:
And yes, the evaluator replaces <TIMESTAMP> with current timestamp, so you dont accidentally override a past evaluation, if you forget to change the dir. I usually use that approach to be safe.